modules/Makefile

```diff
@@ -52,6 +52,7 @@ $(SYS_CTYPES_MODULE_DOC): $(MAKE_SYS_BASIC_TYPES)
 
 
 MODULES_TO_DOCUMENT = \
+	internal/ChapelRange.chpl \
```

Comment:
Now that Ben's change is in, you'll want to add this to `INTERNAL_MODULES_TO_DOCUMENT` instead